storage/innobase/ut/ut0ut.cc

```diff
@@ -58,6 +58,9 @@ this program; if not, write to the Free Software Foundation, Inc.,
 #include "mysql/components/services/log_builtins.h"
 #include "sql/derror.h"
 
+static_assert(std::chrono::steady_clock::is_steady,
```

Comment:
I believe is_steady is more useful to query the properties of all other clocks, here IMHO it's 100% redundant.

Reply:
Actually, no, the reason is that on certain platforms and older glibc versions , the steady_clock/monotonic implementation does not exist and steady_clock is implemented as system_clock.  In that case steady_clock::is_steady will be false.  For our platforms, I have already verified that we are fully covered and added this "just in case" someone tries to build where the real steady_clock implementation is missing.  See https://stackoverflow.com/questions/15032462/c11-clocks-g-steady-clockis-steady-false/15033898#15033898 for details.

Reply:
Great to know, thanks for pointing this out!